docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+
+In this article, I want to explain how UX research drove the development and design of this plugin library and invite you all to follow and contribute as the library continues to improve.
+
+
+## Jobs to be done
+
+After talking to over 60 Gatsby users, I compiled what’s called an “empathy map” in UX design to put myself in the shoes of Gatsby users; basically, this map describes a summary of what the typical Gatsby user experiences when searching for plugins.
+
+![Empathy map](plugin-map.png)
+
+Based on the empathy map above, we identified five “jobs” that Gatsby users want done regarding plugins and packages:
+* Make it easy to search for and find the best, most recent, and relevant plugin for their site.
+    * Scrollable, searchable list of plugins with a search bar at the top, tags attached to each plugin, and accurate information on version compatibility, popularity, maintenance, stability.
+    * Each profile card should have a short description and the profile page should have the Github README file.
+* Make it easy to imitate plugins as they build their own.
+    * The profile cards and pages should easily link to the Github repo.
+* Recommend plugins based on their needs.
+    * You could fill out a profile and login and get suggestions based on the kind of site you’re building, your level of experience, etc.
+    * There could be permanent categories like “for beginners” or other categories.
+    * There can also be changing categories like “editor’s picks,” “trending,” “most popular,” etc.
+* Show (or at least explain) what plugins do (preview) before installation.
+    * Make sure README files are filled out with instructions and a list of sites that use the plugin.
+    * Possible showcase every week?
+* Make it easy to share plugins they build and/or plugins they like.
+    * A huge thank you to @jlengstorf and all involved in building the [Plugin Authoring page in the docs](https://www.gatsbyjs.org/docs/plugin-authoring/#what-files-does-gatsby-look-for-in-a-plugin)
+    * Share buttons from the profile card and/or profile page.
+
+
+## Learning from other plugin libraries
+
+We also analyzed over 10 admirable and/or popular plugin libraries to draw from their strengths and learn from their weaknesses. Examples in no particular order include [JS.coach](https://js.coach/), [VIM Awesome](https://vimawesome.com/), [Best of JS](https://bestof.js.org/), [Sketch extension library](https://sketchapp.com/extensions/), [Chrome plugin library](https://chrome.google.com/webstore/detail/plugins/mmcblfncjaclajmegihojiekebofjcen?hl=en), [Microsoft Visual Studio Code Extensions](https://marketplace.visualstudio.com/VSCode), [Wordpress Plugins](https://wordpress.org/plugins/), [Npms.io](https://npms.io/), [Yarnpkg.com](https://yarnpkg.com/en/packages), [Apple App store](https://www.apple.com/ios/app-store/), which just got redesigned, so I’m interested to learn from their changes, [Google Play store](https://play.google.com/store/apps/top).
```

Comment:
This is tantalizing but feels incomplete. What did we learn from surveying? Were there 3-5 common UI patterns to draw out? What did these do differently? Maybe these tie into your five action items earlier, but this section feels out of place. Possibly we could move some of the features from above into here.